src/main/webapp/dataverseBridgeFragment.xhtml

```diff
@@ -0,0 +1,48 @@
+<ui:composition xmlns:h="http://java.sun.com/jsf/html"
+                xmlns:f="http://java.sun.com/jsf/core"
+                xmlns:ui="http://java.sun.com/jsf/facelets"
+                xmlns:c="http://java.sun.com/jsp/jstl/core"
+                xmlns:p="http://primefaces.org/ui"
+                xmlns:jsf="http://xmlns.jcp.org/jsf">
+    <!-- Dialog DataverseBridge Form -->
+    <p:dialog id="dataverseBridgeDialog" header="Archiving dataset '#{DatasetPage.persistentId}'" widgetVar="tdrForm" modal="true">
+        <p:ajax event="close" update=":datasetForm" />
+        <p:fragment>
+            <h:form class="form-horizontal"  autocomplete="off">
+                <h:outputText value="#{bundle['dataset.archive.dialog.label']}" escape="false"/>
+                <h:panelGrid columns="3" cellpadding="5">
+
+                    <p:outputLabel for="selected-tdr" value="#{bundle['dataset.archive.dialog.label.repository.name']}"/>
+                    <h:selectOneMenu id="selected-tdr"
+                                     value="#{dataverseBridgeDialog.tdrName}"
+                                     required="true" requiredMessage="Please select repository">
```

Comment:
Are requiredMessages not coming from the bundle, or am I missing something here?